app/lib/clients/create_employees.rb

```diff
@@ -5,6 +5,26 @@ def initialize(employees, time_off)
     @dates = create_dates
   end
 
+  def add_employee(employee)
+    EmployeeEvents.create!(external_id: employee['id'], display_name: employee['displayName'],
+                           birthday: employee['birthday'], hire_date: employee['hireDate'],
+                           is_photo_uploaded: employee['isPhotoUploaded'], photo_url: employee['photoUrl'],
+                           time_off_employees: find_time_off_employees)
+  end
+
+  def update_employees
+    @employees.each do |e|
+      if EmployeeEvents.exists?(external_id: e['id'])
+        EmployeeEvents.update(e['id'], display_name: e['displayName'],
+                                       birthday: e['birthday'], hire_date: e['hireDate'],
+                                       is_photo_uploaded: e['isPhotoUploaded'], photo_url: e['photoUrl'],
+                                       time_off_employees: find_time_off_employees)
```

Comment:
There's a bug in this. We're checking for the presence of an `EmployeeEvents` by it's `external_id`, but then trying to update it using its `id`. Those are probably not the same. In any case, I think it would be cleaner to retrieve the record and use it's `update` method instead of using `EmployeeEvents.update`:

```suggestion
      employee_event = EmployeeEvents.find_by_external_id(e['id])
      if employee_event
        employee_event.update(display_name: e['displayName'],
                                       birthday: e['birthday'], hire_date: e['hireDate'],
                                       is_photo_uploaded: e['isPhotoUploaded'], photo_url: e['photoUrl'],
                                       time_off_employees: find_time_off_employees)
```